### ISSUE TITLE
Official Swagger Test Definition Not Working

### ISSUE THREAD

**Issue by BSalita** [jensoleg/swagger-ui#20]
Wonderful project. However, I am unable to get this official Swagger definition to work. Methods are missing.

https://raw.githubusercontent.com/swagger-api/swagger-spec/master/examples/v2.0/json/petstore-expanded.json

Randomly picking Swagger definitions at http://theapistack.com/companies.html shows many are not working either.

**Comment by jensoleg** [comment]
Hi @BSalita 

This project uses the "summary" field to populate the navigation menus and this field is missing in the definition file.
I also noticed that the samples are not rendered correct. This is because the Pet object definition has an embedded reference to the NewPet object. This is not handled yet by this repo. 

I have modified the definition file - adding the summary fields and flatted the objects. Give it a try.

https://gist.githubusercontent.com/jensoleg/b2fe966d16e5152b5ecd/raw/bc8264c57413b3633808450cdecfa7d4acb3ba39/bsalita 

My time is limit at the moment (vacation) but the problem should be fixed. Maybe by someone in the community ?
